webpack/tests/devserver/BUILD.bazel

```diff
@@ -1,19 +1,21 @@
-load("//webpack:defs.bzl", "webpack_dev_server")
+load("@aspect_rules_js//js:defs.bzl", "js_run_devserver")
```

Comment:
So we're completely deleting the devserver rule? Not creating a macro with some defaults set and the word "webpack" in the api?

Reply:
Wasn't planning on it but we could

Reply:
yeah I think since this is a custom ruleset we've already committed to maintaining, it should still have something called `webpack_dev_server`

Reply:
Yeah I think if we have a custom ruleset we shouldn't have to point people elsewhere for a standard devserver that is normally part of webpack, even if it's super simple. Even if it's just a macro to give it a name, although I assume there's some basic defaults that might be nice.

Reply:
cool. I will add a webpack_dev_server macro here then.

Reply:
✅